website/src/components/quickstartTOC/styles.module.css

```diff
@@ -88,10 +90,20 @@ html[data-theme="dark"] .tocList .active span {
     margin-right: auto;
 }
 
+.buttonContainer .prevButton i {
+    font-size: .8rem;
+    margin-right: .4rem;
+}
+
 .buttonContainer .nextButton {
     margin-left: auto;
 }
 
+.buttonContainer .nextButton i {
+    font-size: .8rem;
+    margin-left: .4rem;
+}
+
```

Comment:
When on step 1 of a quickstart, the `Next` button at the bottom of the page has green text with a green button when in dark mode. Once you move to step 2 and on, the button has a dark bg and is readable.
![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/934b0fcd-7b05-4c57-a926-b32e618625ff)